src/plugins/formulas/__tests__/initialization.spec.js

```diff
@@ -0,0 +1,699 @@
+      }).data('handsontable');
+
+      expect(hot1.getDataAtCell(1, 0)).toEqual('hello from sheet 2');
+      expect(hot2.getDataAtCell(0, 0)).toEqual('hello from sheet 1');
+    });
+
+    it('should auto-update the cells referenced between "linked" HOT instances after one of the is modified', () => {
+      const hot1 = handsontable({
+        data: [['hello from sheet 1'], ['=Sheet2!A2']],
+        formulas: {
+          hyperformula: HyperFormula,
+          sheetName: 'Sheet1'
+        },
+        licenseKey: 'non-commercial-and-evaluation'
+      });
+
+      const hot2 = spec().$container2.handsontable({
+        data: [['=Sheet1!A1'], ['hello from sheet 2']],
+        formulas: {
+          hyperformula: getPlugin('formulas').hyperformula,
+          sheetName: 'Sheet2'
+        },
+        licenseKey: 'non-commercial-and-evaluation'
+      }).data('handsontable');
+
+      hot1.setDataAtCell(0, 0, 'Hello from 1');
+
+      expect(hot2.getCell(0, 0).innerText).toEqual('Hello from 1');
+
+      hot2.setDataAtCell(1, 0, 'Hello from 2');
+
+      expect(hot1.getCell(1, 0).innerText).toEqual('Hello from 2');
+    });
+  });
+
+  describe('Registering custom HF settings', () => {
+    it('should register the provided named expressions before creating the HF instance', () => {
+      handsontable({
+        data: [['=MyLocal']],
+        formulas: {
+          hyperformula: HyperFormula,
+          sheetName: 'Sheet1',
+          namedExpressions: [
+            {
+              name: 'MyLocal',
+              expression: '1234'
+            }
+          ]
+        },
+        licenseKey: 'non-commercial-and-evaluation'
+      });
+
+      expect(getDataAtCell(0, 0)).toEqual(1234);
+    });
+
+    it('should register custom function plugins before creating the HF instance', () => {
+      class CustomFP extends FunctionPlugin {
+        customFP() {
+          return 'customFP output';
+        }
+      }
+
+      CustomFP.implementedFunctions = {
+        CUSTOMFP: {
+          method: 'customFP',
+        }
+      };
+
+      handsontable({
+        data: [['=CUSTOMFP()']],
+        formulas: {
+          hyperformula: HyperFormula,
+          sheetName: 'Sheet1',
+          functions: [
+            {
+              name: 'CUSTOMFP',
+              plugin: CustomFP,
+              translations: {
+                enGB: {
+                  CUSTOMFP: 'CUSTOMFP'
+                }
+              }
+            }
+          ]
+        },
+        licenseKey: 'non-commercial-and-evaluation'
+      });
+
+      expect(getDataAtCell(0, 0)).toEqual('customFP output');
+    });
+
+    it('should register a language applying it to the HF instance', () => {
+      handsontable({
+        data: [['TEST'], ['=LITERY.MAŁE(A1)']],
+        formulas: {
+          hyperformula: HyperFormula,
+          sheetName: 'Sheet1',
+          language: {
+            code: 'plPL',
+            language: plPL
+          }
```

Comment:
HF language pack already has language code https://github.com/handsontable/hyperformula/blob/6ec629b16035174d3f63e418d05cd0e98e682eb3/src/i18n/languages/plPL.ts#L407
We can simplify this:

```suggestion
          language: plPL
```